liquibase-core/src/main/java/liquibase/GlobalConfiguration.java

```diff
@@ -188,6 +189,11 @@ public class GlobalConfiguration implements AutoloadedConfigurations {
                 .setDefaultValue(true)
                 .build();
 
+        PRESERVE_SCHEMA_CASE = builder.define("preserveSchemaCase", Boolean.class)
+                .setDescription("Should Liquibase preserve the original case of schemas")
```

Comment:
Maybe describe this as "Should liquibase treat schema and catalog names as case sensitive?"

Reply:
Got it